### PR TITLE
upgrade node version

### PR DESCRIPTION
**Pull request by leggsimon**
@bjfletcher 

This upgrades the node version to 6.9.4, hopefully fixing the master build

### CODE REVIEW

File: test/lib/flagHelper.spec.js [hunk 1]
```diff
@@ -32,38 +32,34 @@ describe('Flags helper', function() {
...
 				expect(flags.flags).to.be.an('array');
-				done();
 			});
 		});
 	});
 	describe('Update Flags', function() {
-		it('Should update the property of a flag', function(done){
+		it('Should update the property of a flag', function(){
 			let origValue;
 			flagHelper.getFlags().then(flags => {
```
Comment (bjfletcher) [inline test/lib/flagHelper.spec.js]: need to return a promise from test here if no done

File: test/lib/flagHelper.spec.js [hunk 2]
```diff
@@ -32,38 +32,34 @@ describe('Flags helper', function() {
...
 					console.log(flags.flags[0].description)
 					expect(flags.flags[0].description).to.not.equal(origValue);
-					done();
 				})
 			});
 		});
-		it('Should increase the version by one', function(done){
+		it('Should increase the version by one', function(){
 			let origVersion;
 			flagHelper.getFlags().then(flags => {
```
Comment (bjfletcher) [inline test/lib/flagHelper.spec.js]: ditto

File: test/lib/flagHelper.spec.js (hunk 1 above)
Comment (leggsimon) [inline test/lib/flagHelper.spec.js]: Ahh, yes, I returned the nested one. Will change

**Comment by bjfletcher** [comment]
@leggsimon apart from the test comments, :+1: How robust are the tests & smoke tests?  If robust, go ahead... if not, some due diligence & preparedness in the event of unforeseen issues :)

**Comment by leggsimon** [comment]
Well I haven't added any new functionality and a look at the tests look like they cover it well enough...

**Pull request merged**